lib/src/model/movie_result.dart

```diff
@@ -43,10 +45,11 @@ class MoviesResult {
       voteCount: parsedJson['vote_count'],
       id: parsedJson['id'],
       video: parsedJson['video'],
-      voteAverage: parsedJson['vote_average'],
       title: parsedJson['title'],
       popularity: parsedJson['popularity'],
-      posterPath: parsedJson['poster_path'],
+      posterPath: parsedJson['poster_path'] != null
+          ? MovieStrings.imagesPath + parsedJson['poster_path']
+          : MovieStrings.defaultImage,
```

Comment:
Take a look at this null operator: https://news.dartlang.org/2015/08/dart-112-released-with-null-aware.html, this can make you save coding time a it will make your code easy to read

Reply:
I took a look at it. It was useful for me to learn these types of operators but in this case I cannot use any of those because  the expression will never resolve null.